docs/milestones.md

```diff
@@ -0,0 +1,66 @@
+# Milestones 
+
+---
+
+## [M0] Documento con descripción de las entidades:
+
+- **Producto a entregar:** 
+  Un documento que describa las entidades clave (población, edades, natalidad, mortalidad, migración) y sus relaciones, sirviendo como base conceptual para estructurar los datos demográficos de la Sierra de Segura.
+
+- **Viabilidad:** 
+  Este milestone será viable cuando el documento incluya una descripción clara de las entidades y relaciones necesarias para recopilar y organizar los datos demográficos de manera efectiva.
+
+---
+
+## [M1] Recopilación de conjunto de datos demográficos:
```

Comment:
¿Has considerado integrar este milestone dentro del m2? La recolección de información se considera una tarea mas que un hito en el desarollo aun siendo necesario realizarlo not supone la con la suficiente importancia para que tenga un milestoen propio